tools/pd-ctl/pdctl/command/store_command.go

```diff
@@ -181,6 +193,57 @@ func deleteStoreCommandFunc(cmd *cobra.Command, args []string) {
 	cmd.Println("Success!")
 }
 
+func deleteStoreCommandByAddrFunc(cmd *cobra.Command, args []string) {
+	if len(args) != 1 {
+		cmd.Usage()
+		return
+	}
+	addr := args[0]
+
+	// fetch all the stores
+	r, err := doRequest(cmd, storesPrefix, http.MethodGet)
```

Comment:
emm....can we add an `API` in the server to find the store ID by the store address? maybe it's more general needs. cc @disksing

Reply:
It is OK for me to add an `API`, meanwhile, we should consider that it is not an efficient `API` because of iteration to find a store ID.

Reply:
I think both way is fine. There won't be too many stores.